src/Microsoft.Health.Dicom.Core/Exceptions/ElementValidationException.cs

```diff
@@ -5,24 +5,31 @@
 
 using EnsureThat;
 using FellowOakDicom;
+using Microsoft.Health.Dicom.Core.Extensions;
 using Microsoft.Health.Dicom.Core.Features.Validation;
 
 namespace Microsoft.Health.Dicom.Core.Exceptions
 {
     public class ElementValidationException : ValidationException
     {
-        public ElementValidationException(string name, DicomVR vr, ValidationErrorCode errorCode, string message) : base(message)
-        {
-            Name = EnsureArg.IsNotNull(name, nameof(name));
-            VR = EnsureArg.IsNotNull(vr, nameof(vr));
-            ErrorCode = EnsureArg.EnumIsDefined(errorCode, nameof(errorCode));
-        }
+        public ElementValidationException(string name, DicomVR vr, ValidationErrorCode errorCode)
+            : this(name, vr, null, errorCode)
+        { }
 
-        public ElementValidationException(string name, DicomVR vr, string value, ValidationErrorCode errorCode, string message) : base(message)
+        public ElementValidationException(string name, DicomVR vr, string value, ValidationErrorCode errorCode)
+            : this(name, vr, value, errorCode, errorCode.GetMessage())
+        { }
+
+        public ElementValidationException(string name, DicomVR vr, ValidationErrorCode errorCode, string message)
+            : this(name, vr, null, errorCode, message)
+        { }
+
+        public ElementValidationException(string name, DicomVR vr, string value, ValidationErrorCode errorCode, string message)
+            : base(message)
         {
             Name = EnsureArg.IsNotNull(name, nameof(name));
             VR = EnsureArg.IsNotNull(vr, nameof(vr));
-            Value = EnsureArg.IsNotNull(value, nameof(value));
+            Value = value;
```

Comment:
intended to remove Null check on value?

Reply:
Yeah, as the value seems to really be optional. It also makes it easier to chain the constructors if we allow `null` in both ctor calls lol